cocos/network/WebSocket.cpp

```diff
@@ -574,16 +597,33 @@ int WebSocket::onSocketCallback(struct libwebsocket_context *ctx,
                         	if (remaining != n)
                         		writeProtocol |= LWS_WRITE_NO_FIN;
                         }
-
+	#if(CC_TARGET_PLATFORM != CC_PLATFORM_OHOS)
                         auto bytesWrite = libwebsocket_write(wsi,  &buf[LWS_SEND_BUFFER_PRE_PADDING], n, (libwebsocket_write_protocol)writeProtocol);
-
+	
                         // Buffer overrun?
                         if (bytesWrite < 0)
                         {
                             //break;
+                       }
+					       // Do we have another fragments to send?
+					    else if (remaining != n)
+                        {
+                            data->issued += n;
+                            //break;
                         }
-                        // Do we have another fragments to send?
-                        else if (remaining != n)
+                        // Safely done!
+                        else
+                        {
+                            CC_SAFE_DELETE_ARRAY(data->bytes);
+                            CC_SAFE_DELETE(data);
+                            CC_SAFE_DELETE_ARRAY(buf);
+                            _wsHelper->_subThreadWsMessageQueue->erase(iter++);
+                            CC_SAFE_DELETE(subThreadMsg);
+                        }
+					   
+	#else
+	   					 // Do we have another fragments to send?
+						 if (remaining != n)
```

Comment:
Wrong indent.

Reply:
Modified

Reply:
`if (remaining != n)` still has wrong indent.

Reply:
Modified，Please check it